src/modules/liking.js

```diff
@@ -0,0 +1,63 @@
+import { countComments } from './counters.js';
+
```

Comment:
- [ ] [OPTIONAL] Kindly extract the calling API logic in an external module instead of mixing dom code with API calling and use it here to make your code more cleaner and organized.